modules/order/src/Entity/Order.php

```diff
@@ -458,6 +480,7 @@ public function postSave(EntityStorageInterface $storage, $update = TRUE) {
     // If no order number has been set explicitly, set it to the order ID.
     if (!$this->getOrderNumber()) {
       $this->setOrderNumber($this->id());
+      $this->setRefreshState(self::REFRESH_SKIP);
```

Comment:
We set it as `REQUEST_SKIP` because order storage will then set it as `NULL` correct?